file/reader.go

```diff
@@ -204,6 +207,9 @@ func GetStateFromFile(filename string) (*state.KongState, error) {
 
 func readFile(kongStateFile string) (*fileStructure, error) {
 
+	if kongStateFile == "-" {
+		return readStdin()
```

Comment:
This elaborate method call can be simply replaced by:
`data, err = ioutil.ReadAll(os.Stdin)` and then the logic below will take care of unmarshaling it into a YAML.

Let's not duplicate the YAML Unmarshaling logic as well.